sparrow_datums/boxes/boxes_test.py

```diff
@@ -109,3 +110,19 @@ def test_resize():
     assert new_boxes.image_width == 32
     assert new_boxes.image_height == 16
     np.testing.assert_equal(new_boxes.array, np.array([8, 4, 8, 4]))
+
+
+# Make this test work after Ben gets back. Also, replicate for the keypoint class.
```

Comment:
Will you delete these comments? I think we can safely skip this test for now.

Reply:
done